docs/src/app/components/pages/components/refresh-indicator.jsx

```diff
@@ -0,0 +1,82 @@
+let React = require('react');
+let { RefreshIndicator } = require('material-ui');
+let ComponentDoc = require('../../component-doc');
+
+
+let RefreshIndicatorPage = React.createClass({
+
+  render() {
+
+    let code =
+      '// Ready status \n' +
+      '<RefreshIndicator percentage={30} size={40} left={10} top={5} status="ready" />\n' +
+      '<RefreshIndicator percentage={60} size={40} left={10} top={5} status="ready" />\n' +
+      '<RefreshIndicator percentage={80} size={40} left={10} top={5} status="ready" />\n' +
+      '<RefreshIndicator percentage={100} size={40} left={10} top={5} status="ready" />\n' +
+      '// Loading status \n' +
+      '<RefreshIndicator size={40} left={80} top={5} status="loading" />\n';
+
+    let componentInfo = [
+      {
+        name: 'Props',
+        infoArray: [
+          {
+            name: 'size',
+            type: 'number',
+            header: 'default: 40',
+            desc: 'Size in pixel.'
+          },
+          {
+            name: 'percentage',
+            type: 'number',
+            header: 'default: 0',
+            desc: 'The confirmation progress to fetch data. Max value is 100'
```

Comment:
According to the design specs, the refresh arrow doesn't seem to indicate percent complete progress - rather, it's just part of a swipe animation - https://www.google.com/design/spec/patterns/swipe-to-refresh.html#swipe-to-refresh-swipe-to-refresh

I'm not sure if this actually changes the implementation of this component since we can still use this prop to implement the swipe animation. However, it doesn't change how we explain this prop and what we name it.